src/ui/components/DropdownMenu/styles.scss

```diff
@@ -6,8 +6,8 @@
   @include text-align-start();
 
   display: inline-block;
-  padding-bottom: 6px;
-  padding-top: 8px;
+  padding-bottom: 2px;
+  padding-top: 4px;
```

Comment:
I still have a different between the menu and login button. It seems there is a `0.25px` difference. Could you set the height of the `DropdownMenu` to `27px`?

Reply:
@willdurand , I was looking at desktop version where `height: 27px` had no effect :(
Then I saw the mobile version :P , I forgot #mobileFirst
Here you go -
Mobile:
![mobile](https://user-images.githubusercontent.com/5318732/30819584-d0452aa2-a23c-11e7-9d06-6ce6b51657b6.gif)


Desktop:
![ezgif-4-2d44bd1686](https://user-images.githubusercontent.com/5318732/30818991-e6c4d6bc-a23a-11e7-9f8d-4e256596c80e.gif)

Reply:
Yeah, sorry for that. There is actually an issue on desktop now :-(